src/components/tabController/TabPage.tsx

```diff
@@ -49,9 +51,14 @@ export default function TabPage({
   useAnimatedReaction(() => {
     return targetPage.value === index;
   },
-  isActive => {
+  (isActive, wasActive) => {
```

Comment:
I think `wasActive` is actually `previousActive` or `previousActiveState`

Reply:
Not sure I understand? 
You mean the name is not good? 
what's the different between wasActive and previousActive?

Reply:
It was a little confusing to me, not sure why. I think it's more standard to call it `prev` or `previous`.
The API for `useAnimatedReaction` is:
```
export function useAnimatedReaction<D>(
  prepare: () => D,
  react: (prepareResult: D, preparePreviousResult: D | null) => void,
  deps?: DependencyList
): void;
```

This is also the case in other methods, but if you think it's more clear then I'm fine with it.